### PR TITLE
Segment Datalakes Azure Document Changes.

### PR DESCRIPTION
**Pull request by hrdamani**

### CODE REVIEW

**Comment by Lfdelossantos** [comment]
Change Step 6 bullet point 1 and 2. They shouldn't be sep steps.  
Maybe something like this 

1. Connect to a [Hive metastore](https://docs.databricks.com/data/metastores/external-hive-metastore.html) on your Databricks cluster. The easiest way to do that is by copying the following Spark configuration...

File: src/connections/storage/catalog/data-lakes/index.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ redirect_from: '/connections/destinations/catalog/data-lakes/'
 
 
 Segment Data Lakes provide a way to collect large quantities of data in a format that's optimized for targeted data science and data analytics workflows. You can read [more information about Data Lakes](/docs/connections/storage/data-lakes/) and learn [how they differ from Warehouses](/docs/connections/storage/data-lakes/comparison/) in Segment's Data Lakes documentation.
+Segment supports two type of data-lakes as of now:
```
Comment (markzegarelli) [inline src/connections/storage/catalog/data-lakes/index.md:9]:
```suggestion
Segment supports two type of data-lakes:
```

File: src/connections/storage/catalog/data-lakes/index.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ redirect_from: '/connections/destinations/catalog/data-lakes/'
...
+- [AWS Data Lakes](https://segment.com/docs/connections/storage/catalog/data-lakes/#set-up-segment-data-lakes)
```
Comment (markzegarelli) [inline src/connections/storage/catalog/data-lakes/index.md:10]:
```suggestion
- [AWS Data Lakes](/docs/connections/storage/catalog/data-lakes/#set-up-segment-data-lakes)
```

File: src/connections/storage/catalog/data-lakes/index.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ redirect_from: '/connections/destinations/catalog/data-lakes/'
...
+- [Azure Data Lakes](https://segment.com/docs/connections/storage/catalog/data-lakes/#set-up-azure-data-lakes)
```
Comment (markzegarelli) [inline src/connections/storage/catalog/data-lakes/index.md:11]:
```suggestion
- [Azure Data Lakes](/docs/connections/storage/catalog/data-lakes/#set-up-azure-data-lakes)
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**